deployer/README.md

```diff
@@ -2,9 +2,9 @@
 
 The Yari deployer does two things. First, it's used to upload document
 redirects, pre-built document pages, static files (e.g. JS, CSS, and
-image files), and sitemap files into an existing AWS S3 bucket.
-Since we serve MDN from a S3 bucket via a CloudFront CDN, this is the
-way we upload a new version of the site.
+image files), and sitemap files into an existing AWS S3 bucket. Since
+we serve MDN document pages from a S3 bucket via a CloudFront CDN,
```

Comment:
Interestingly(?), I use Grammarly and they recommend using `an` when the next word starts with an `s`.

Reply:
I totally agree with Grammarly, and I sometimes I remember to do that, but I'm not consistent! Thanks.